lib/Controller/WopiController.php

```diff
@@ -714,4 +729,19 @@ public function getTemplate($fileId, $access_token) {
 		}
 	}
 
+	/**
+	 * Check if the encryption module uses a master key.
+	 *
+	 * @return bool
+	 */
+	private function isMasterKeyEnabled()
+	{
```

Comment:
```suggestion
	/**
	 * Check if the encryption module uses a master key.
	 */
	private function isMasterKeyEnabled(): bool {
```

Very small nitpick on the coding style and we can actually type hint on the master branch (and for the stable 4.x releases)

Otherwise :+1: from my side.